### ISSUE TITLE
Errors when running train.py

### ISSUE THREAD

**Issue by hugeinteger** [eveningglow/fast-style-transfer-pytorch#1]
1. I guess there is a missing argument (save_entry) in arguments parsing, I've added lines below in 'train.py':
parser.add_argument('--save_every', type=int, default=5000,
                                    help='How often do you want to save result?')
2. What should be trn-dir for running train.py?

python train.py --trn_dir='data/train' --style_path='style/abstract_1.png', --lambda_s=500000

Seems it should be downloaded like below:

mkdir -p data/train
cd data/train
wget http://www.vlfeat.org/matconvnet/models/beta16/imagenet-vgg-verydeep-19.mat
wget http://msvocds.blob.core.windows.net/coco2014/train2014.zip
unzip train2014.zip

**Comment by lyupei** [comment]
yes, you should add this argument and some CODE or it will NOT save weights until the end of train.

the trn_dir should be a directory containing some image folders or it will not find any images to use, such as coco2014/train_images/cocoxxxx.jpg you are supposed to pass coco2014 to this argument.

**Comment by hugeinteger** [comment]
Is it possible to train a model for a new style image on a regular machine without GPUs?

**Comment by lyupei** [comment]
It may take several weeks...
